include/swift/AST/NameLookup.h

```diff
@@ -36,52 +37,91 @@ namespace swift {
   class TypeDecl;
   class ValueDecl;
   struct SelfBounds;
+  class PatternBindingInitializer;
+  class DefaultArgumentInitializer;
+  class NominalTypeDecl;
+
+  namespace ast_scope {
+  class ASTSourceFileScope;
+  class ASTScopeImpl;
+  } // namespace ast_scope
+
+  /// LookupResultEntry - One result of unqualified lookup.
```

Comment:
Why are you indenting this out, that just increases the diff without need.

Reply:
Probably a git-clang-format thing--I'll fix.

Reply:
I think it's fixed. Let me know if not. I have tried to git-format it all but may have missed something.